tests/test_ephys.py

```diff
@@ -0,0 +1,229 @@
+"""Tests parsing of session information from ephys rig."""
+
+import csv
+import json
+import os
+import unittest
+from pathlib import Path
+from xml.dom import minidom
+
+from aind_data_schema.core.session import Session
+from aind_data_schema.models.modalities import Modality
+
+from aind_metadata_mapper.ephys.session import EphysEtl
+
+RESOURCES_DIR = (
+    Path(os.path.dirname(os.path.realpath(__file__))) / "resources" / "ephys"
+)
+EXAMPLE_STAGE_LOGS = [
+    RESOURCES_DIR / "newscale_main.csv",
+    RESOURCES_DIR / "newscale_surface_finding.csv",
+]
+EXAMPLE_OPENEPHYS_LOGS = [
+    RESOURCES_DIR / "settings_main.xml",
+    RESOURCES_DIR / "settings_surface_finding.xml",
+]
+
+EXPECTED_SESSION = RESOURCES_DIR / "ephys_session.json"
+
+
+class TestSchemaWriter(unittest.TestCase):
+    """Test methods in SchemaWriter class."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Load record object and user settings before running tests."""
+        # TODO: Add visual stimulus
```

Comment:
@micahwoodard is all of `experiment_data` the information that isn't contained in the openephys logs/files, i.e. the stuff we need tracked in SLIMS?

Reply:
Yes I couldn't find that info in the stage or openephys log so I think it will have to come from slims. One thing I'm not super sure on is the targeted_ccf_coordinates. Is there a standard lookup table based on the primary_targeted_structure or a translation between the stage coordinates or something?